src/router_utils.erl

```diff
@@ -565,6 +565,12 @@ trace(DeviceID) ->
         ],
         debug
     ),
+    _ = erlang:spawn(fun() ->
+        Timeout = application:get_env(router, device_trace_timeout, 240),
```

Comment:
Will anything happen if this is a string?

Reply:
this is meant to be change manually it is not exposed